controllers/postController.js

```diff
@@ -0,0 +1,69 @@
+import * as postService from '../services/postService.js';
+
+export const create = async (req, res) => {
+  try {
+    const { title, content } = req.body;
+    const post = await postService.create({ title, content });
+    res.status(201).json({ data: post });
+  } catch (err) {
+    res.status(500).json({ error: 'add post failed' });
+  }
+};
+
+export const getAll = async (req, res) => {
+  try {
+    const posts = await postService.getAll();
+    res.status(200).json({ data: posts });
+  } catch (err) {
+    res.status(500).json({ error: 'get all posts failed' });
+  }
+};
+
+export const getById = async (req, res) => {
+  try {
+    const post = await postService.getById(req.params.postId);
+    if (!post) {
+      let err = new Error('post not found');
+      err.status = 404;
+      throw (err);
+    }
+    res.status(200).json({ data: post });
+  } catch (err) {
+    res.status(err.status || 500).json({ error: err.message || 'find post by id failed' });
+  }
+};
+
+export const update = async (req, res) => {
+  try {
+    const { postId } = req.params;
+    const { title, content } = req.body;
+    const result = await postService.update(postId, { title, content });
```

Comment:
```suggestion
    const result = await postService.update( {_id: postId, title, content });
```

요런식으로 써 보는건 어떨까요??

postService에서도 nextPost 요런식으로 받아서 

```javascript
const result = await Post.updateOne({
    _id: nextPost._id
  }, nextPost);
```

요런식으로 쓸 수 있을거 같습니당!

(요건 옵셔널한 코멘트라 소형님이 판단해주세요!)

Reply:
아 네 PUT 이니까 그게 맞네요! 수정하겠습니다!!